app/src/main/java/com/app/roman/mob_dev/RetrofitClient.java

```diff
@@ -0,0 +1,26 @@
+package com.app.roman.mob_dev;
+
+import android.app.Application;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitClient extends Application {
+
+    private static Retrofit retrofit = null;
+    private static final String BASE_URL = "https://simplifiedcoding.net/demos/";
+
+    private static Retrofit getRetrofitInstance() {
```

Comment:
You should avoid using static methods in Application class. You should access application via `getContext().getApplication()` instead